src/common/styles/colors.scss

```diff
@@ -55,6 +55,7 @@
     --neutral-alpha-60: rgba(0, 0, 0, 0.6);
     --neutral-alpha-70: rgba(0, 0, 0, 0.7);
     --neutral-alpha-80: rgba(0, 0, 0, 0.8);
+    --neutral-alpha-90: rgba(0, 0, 0, 0.8);
```

Comment:
value is same as previous line (0.8 alpha), is that intended?

Reply:
it was a mistake. good catch.